docs/transpile/qiskit-transpiler-service.mdx

```diff
@@ -69,7 +69,8 @@ The following examples demonstrate how to transpile circuits using the Qiskit tr
     )
     transpiled_circuit = cloud_transpiler_service.run(random_circ)
     ```
-​## Next steps
+
+## Next steps
```

Comment:
This was a really weird one. Not sure why the text as originally written did not properly render as a heading, but only by completely deleting the line and starting it over did it render properly. 🤷

Reply:
Because there wasn't a blank line between the code block and the header, so markdown was failing to recognize it correctly. Technically headers should always have one blank line above and below them.

Reply:
Believe it or not but I tried that... it was as if there was an invisible character somewhere in there. Which might have been the case, actually.